### PR TITLE
[PTX-21401] Add EKS Scheduler Driver

### PR DESCRIPTION
**Pull request by kphalgun-px**
**What this PR does / why we need it**:
This PR implements EKS scheduler driver to perform cluster upgrade.

**Which issue(s) this PR fixes** (optional)
Closes #PTX-21401

**Special notes for your reviewer**:
I will update once the test run is complete.

### CODE REVIEW

**Review by spai-px — APPROVED** [review]
LGTM

File: deployments/deploy-ssh.sh [hunk 1]
```diff
@@ -763,6 +763,12 @@ spec:
       value: "${PX_BACKUP_MONGODB_PASSWORD}"
     - name: ENABLE_GRAFANA
       value: "${ENABLE_GRAFANA}"
+    - name: EKS_CLUSTER_NAME
```
Comment (lsrinivas-pure) [inline deployments/deploy-ssh.sh:766]: can you try if these values can get populated automatically

Comment (kphalgun-px) [reply deployments/deploy-ssh.sh:766]: Sure.

**Comment by kphalgun-px** [comment]
Thank you for reviewing. I have raised another PR https://github.com/portworx/torpedo/pull/2158 for these changes.

**Pull request closed without merging**